src/app/CharactersModel.js

```diff
@@ -0,0 +1,12 @@
+import httpClient from './AxiosHttpClient';
+
+const CHARACTERS_ROUTE = 'characters';
+
+export default {
+  async getAll() {
+    const GET_ALL_CHARACTERS_ROUTE = `${CHARACTERS_ROUTE}/getAll`;
+    const response = await httpClient.get(GET_ALL_CHARACTERS_ROUTE);
+
+    return response.data;
+  },
+};
```

Comment:
This isn't a **model**, maybe a `service`.

Our model **has to be** the representation of _something_, like that:
![image](https://user-images.githubusercontent.com/34130446/67504896-9e060a00-f660-11e9-8da4-e92537f68b55.png)


In our case, a **Character Representation**...
You can't forget that *"fill()"* **method** that our model, like that:
![image](https://user-images.githubusercontent.com/34130446/67504869-92b2de80-f660-11e9-8255-8dd9af45a22b.png)